Block/Info/Cc.php

```diff
@@ -31,15 +31,8 @@ public function getCcTypeName()
 
         if (isset($types[$ccType])) {
             return $types[$ccType]['name'];
-        }
-        // TODO::Refactor this block after tokenization of the alternative payment methods.
-        // This elseif block should be removed after the tokenization of the alternative payment methods (In progress: PW-6764). More general approach is required.
-        // Also remove `sepadirectdebit` from translation files.
-        elseif ($ccType == 'sepadirectdebit') {
-            return __('sepadirectdebit');
-        }
-        else {
-            return __('Unknown');
+        } else {
+            return $ccType;
```

Comment:
As the TODO says, can we remove sepadirectdebit from the translation files now? If it isn't being used anywhere else as a string shown to the user I'd say let's remove it in this PR

Reply:
After talking with @candemiralp we decided that it is a solution that it is useful in case that there is not mapping. Furthermore for now it would be nice to keep the list up to date with apple pay

Reply:
@acampos1916 Could you review again ?